src/js/ui/menu.js

```diff
@@ -308,7 +308,7 @@ Menu.prototype.item = function(sectionIndex, itemIndex, item) {
 
 Menu.prototype.selection = function(callback_or_sectionIndex, itemIndex) {
   if (typeof callback_or_sectionIndex === 'function') {
-    this._selections.push(callback);
+    this._selections.push(callback_or_sectionIndex);
```

Comment:
This is another a pre-existing problem, not your fault, but do you think you can change this to the following form, and fixing up the references in the function?
```javascript
Menu.prototype.selection = function(sectionIndex, itemIndex) {
  var callback;
  if (typeof sectionIndex === 'function') {
    callback = sectionIndex;
    sectionIndex = undefined;
  }
```

I eventually would do this myself, but I don't want to overwrite your contribution. I admit it's verbose, but I prefer sane variable names over less lines of code.